docs/commands/rhoas_service-registry_rule.md

```diff
@@ -4,22 +4,23 @@ Manage artifact rules in a Service Registry instance
 
 ### Synopsis
 
-Set of commands for managing validity and compatibility rules for an entire Service Registry instance or individual for each artifact.
-Service Registry rules govern how artifact content can evolve over time by validating for any artifact updates
+Configure the validity and compatibility rules that govern artifact content.
 
-Rules can be configured as global rules for entire Service Registry instances or specific artifacts.
+When you add or update an artifact, Service Registry applies rules to check the validity and compatibility of the artifact content. Artifact-specific rules apply to the specified artifact only. Global rules apply to all artifacts in a particular Service Registry instance. Configured artifact-specific rules override any configured global rules. Before a new artifact version can be uploaded to the registry, all configured global rules or artifact-specific rules must pass.
```

Comment:
Suggest "Artifact rules apply to the specified artifact only" for minimalism and to match the current UI and docs. 

When we've defined artifact rules as applying to the specified artifact only, I don't think we need to call them "artifact-specific rules" elsewhere in the doc.